### ISSUE TITLE
Some images returned in API are not present in source papers.

### ISSUE THREAD

**Issue by davenquinn** [UW-COSMOS/Cosmos#136]
I did a search within the new IODP set for COSMOS, through the xDD API, for the “Maastrichtian” geologic period which abuts the K-T boundary. Results can be viewed [here](https://xdddev.chtc.io/set_visualizer/sets/iodp?query=Maastrichtian&type=Figure) This brings back a lot of good data, but one interesting thing is that starting at about result #10 and continuing for 8 or so iterations, it appears that we are returning duplicate figures; these are attributed to different papers. For at least one of the papers, the figure is not included in the pdf version of the paper.

Perhaps things are getting shuffled around in the pipeline, or there's something weird going on with how IODP journal articles are being split for ingestion.

Here's API links for some of the offending objects:
https://xdd.wisc.edu/sets/iodp/cosmos/api/object/a74d37aff204b26ba74ffc3abd472ed67ffc1486
https://xdd.wisc.edu/sets/iodp/cosmos/api/object/0fa4bdb330241854711b83be10bb453fda026851
https://xdd.wisc.edu/sets/iodp/cosmos/api/object/0b115ea35ed2f41828ea857a2aff4dadf5ebf262
https://xdd.wisc.edu/sets/iodp/cosmos/api/object/24a4846d2fd59f7544db53bc06d7cbc7dd261ac5

And here's the figure in questions:
![K-T-core](https://user-images.githubusercontent.com/1537910/99134206-5abb6800-25e2-11eb-8b41-441bf1d4cf62.png)

**Comment by iross** [comment]
Ahhhh good catch. It's an xDD issue. I'll open an issue over there Monday.
Looks when fetching was done, I kept saving things with a name of
"Chapter8.pdf" to the same place on disk. So all things with a (in IODP)
name of Chapter8.pdf all pointed to the same file on disk, which kept
getting overwritten. See:

https://xdd.wisc.edu/api/articles?docid=5e111254998e17af825a089c&fields=filepath,file_urls
https://xdd.wisc.edu/api/articles?docid=5e0fc7ce998e17af8259f5a9&fields=filepath,file_urls

I've since added protections against this kind of thing and refetched for
the impacted source, not realizing IODP was impacted as well.
